sdk/typescript/src/resources/component/componentVersionsListSample.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import {
+  ComponentVersionsListOptionalParams,
+} from "@azure/arm-machinelearning";
+
+import { client, getEnvironmentVariable } from "../../utils";
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const resourceGroupName = getEnvironmentVariable("RESOURCEGROUP_NAME");
+const workspaceName = getEnvironmentVariable("WORKSPACE_NAME");
+
+/**
+ * This sample demonstrates how to List component versions.
+ *
+ * @summary List component versions.
+ * x-ms-original-file: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-10-01/examples/ComponentVersion/list.json
+ */
+async function listComponentVersion() {
+  const name = "command_component_basic";
+  const orderBy = "createdtime desc";
+  const top = 1;
+  const options: ComponentVersionsListOptionalParams = { orderBy, top };
+  const resArray = new Array();
```

Comment:
Could you add type for the `resArray`? Something like `const resArray = new Array<Item>();`

Reply:
Thanks, updated